docs/src/HowToGuides/Atmos/MoistureAndPrecip.md

```diff
@@ -69,27 +69,8 @@ using CLIMAParameters
 using CLIMAParameters.Planet
 using CLIMAParameters.Atmos.Microphysics
 
```

Comment:
We should probably also update the documentation text above, since our structure is much simpler now. Maybe something like:

The microphysics parameterizations used to compute the sources and sinks for
moisture and precipitation variables  depends on 
[CLIMAParameters.jl](https://github.com/CliMA/CLIMAParameters.jl).
A set with all the parameter values needs to be created and passed in as one
of the arguments.